ftw/upgrade/workflow.py

```diff
@@ -151,9 +152,15 @@ def _migrate_entry(entry):
 
 class WorkflowSecurityUpdater(object):
 
-    def update(self, changed_workflows, reindex_security=True):
+    def update(self, changed_workflows, reindex_security=True, savepoints=1000):
         types = self.get_suspected_types(changed_workflows)
-        for obj in self.lookup_objects(types):
+
+        if savepoints:
+            objects = SavepointIterator.build(self.lookup_objects(types), savepoints)
```

Comment:
You can always build the savepoint iterator; this if/else-block is part of the build function.
See: https://github.com/4teamwork/ftw.upgrade/blob/ba01789468b7f3c756aae6624a1ceb0c50b93d0e/ftw/upgrade/utils.py#L140-L143

Reply:
Nice catch, did not see that. Updated.